modules/data/panel_ui.components.json

```diff
@@ -143,6 +143,17 @@
         "selectorData": "toolbarbutton",
         "strategy": "tag",
         "groups": []
-    }
+    },
 
+    "panel-ui-bookmarks": {
+        "selectorData": "appMenu-bookmarks-button",
+        "strategy": "id",
+        "groups": []
+    },
+
+    "bookmark-by-title": {
```

Comment:
I think it's possible that the previous version of this was finding another element on the page, and that wasn't a problem for fast computers but it was a problem for the GHA windows machine, which is very slow

Reply:
I struggled with that selector and I were sure that there is a better way to write that selector.